src/main/features/agreement/views/lot.njk

```diff
@@ -6,30 +6,16 @@
     
 <div class="govuk-breadcrumbs">
   <ol class="govuk-breadcrumbs__list">
-  
-    
     <li class="govuk-breadcrumbs__list-item">
       <a class="govuk-breadcrumbs__link" href="/">Home</a>
     </li>
-    
-  
-    
     <li class="govuk-breadcrumbs__list-item">
       <a class="govuk-breadcrumbs__link" href="/agreement?id=RM6263">Digital Specialists and Programmes</a>
     </li>
-    
-  
-    
-    <li class="govuk-breadcrumbs__list-item" aria-current="page">Lot 1: Programmes </li>
```

Comment:
need to make the as parametrise something like below. Don't hard code page title as Lot 1 it's should be dynamic based on query parameter.

`{{agreement_lot.number}} Programmes `

Reply:
This is what is being deleted, right below you can find that it is dynamic.